lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -135,6 +137,12 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     display: none;
   }
 
+  // used for the footer - hidden content still takes up space when sidebar is collapsed
+  .nx-global-sidebar__hide-when-collapsed {
```

Comment:
Is this intended to be used by custom content in downstream apps? If not I'm not sure it should be it's own class rather than just these properties applied to the right elements in the right circumstances.

Reply:
> Is this intended to be used by custom content in downstream apps? If not I'm not sure it should be it's own class rather than just these properties applied to the right elements in the right circumstances.

My intent was that it would be available to us, I did not document it.

Reply:
> My intent was that it would be available to us, I did not document it.

But in that case why does it exist? Why not just style the existing classes as needed?

Reply:
> > My intent was that it would be available to us, I did not document it.
> 
> But in that case why does it exist? Why not just style the existing classes as needed?

One way I add multiple identical attributes to multiple classes, this way I create one class with those attributes. Then if for some reason we need this behaviour for something else we have it.

We have utility JS we chose not to expose, why not have utility CSS we chose not to expose?